LenovoLegionToolkit.WPF/Controls/Automation/AutomationPipelineControl.cs

```diff
@@ -290,6 +290,9 @@ private string GenerateSubtitle()
         if (AutomationPipeline.Trigger is IWiFiConnectedPipelineTrigger wt && wt.Ssids.Any())
             result += $" | {string.Join(",", wt.Ssids)}";
 
+        if (AutomationPipeline.Trigger is IPeriodicAutomationPipelineTrigger pet && pet.PeriodMinutes != null)
+            result += $" | {string.Join(": ", Resource.PeriodicActionPipelineTriggerTabItemContent_PeriodMinutes, pet.PeriodMinutes.ToString())}";
```

Comment:
String interpolation would be enough here:

`result += $" | {Resource.PeriodicActionPipelineTriggerTabItemContent_PeriodMinutes}: {pet.PeriodMinutes}";`